src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx

```diff
@@ -73,8 +73,8 @@ const SaleHeaderBanner = forwardRef<HTMLDivElement, SaleHeaderBannerProps>(
     ref,
   ) => {
     const { className: baseClassName, ...commonProps } = getCommonProps(props, 'SaleHeaderBanner');
-    const isOpenForBidding = auctionState === AuctionState.openForBidding;
-    const isClosed = auctionState === AuctionState.past;
+    const isOpenForBidding = auctionState === AuctionStatus.live;
+    const isClosed = auctionState === AuctionStatus.past;
```

Comment:
I could make the same argument here, that we might have this in Remix rather than seldon...

Reply:
Not sure if having an enum is reason to move to remix. We can use the enum to pass in here. I am mirroring the values that will be passed back from the API for AuctionStatus so if they differ we will probably want to update that enum as well.